### PR TITLE
Display quick replies option on external channel claim/config

### PR DESCRIPTION
**Pull request by allancslima**
This PR is related to enabling quick replies on Courier's external channel: https://github.com/nyaruka/courier/pull/275

### CODE REVIEW

File: temba/channels/models.py [hunk 1]
```diff
@@ -257,7 +257,10 @@ class Channel(TembaModel):
     CONFIG_SEND_METHOD = "method"
     CONFIG_SEND_BODY = "body"
     CONFIG_MT_RESPONSE_CHECK = "mt_response_check"
-    CONFIG_DEFAULT_SEND_BODY = "id={{id}}&text={{text}}&to={{to}}&to_no_plus={{to_no_plus}}&from={{from}}&from_no_plus={{from_no_plus}}&channel={{channel}}"
+    CONFIG_DEFAULT_SEND_BODY = (
+        "id={{id}}&text={{text}}&to={{to}}&to_no_plus={{to_no_plus}}&from={{from}}&from_no_plus={{from_no_plus}}"
+        "&channel={{channel}}{{quick_replies}}"
+    )
```
Comment (nicpottier) [inline temba/channels/models.py]: I'm fine documenting that people can use the `{{quick_replies}}` variable, but how about we leave it out of the default?

Comment (allancslima) [reply temba/channels/models.py]: I agree.

File: templates/channels/types/external/claim.haml [hunk 2]
```diff
@@ -27,18 +27,27 @@
...
     %pre<
-      http://myservice.com/send.php?from={% templatetag openbrace %}{% templatetag openbrace %}from{% templatetag closebrace %}{% templatetag closebrace %}&text={% templatetag openbrace %}{% templatetag openbrace %}text{% templatetag closebrace %}{% templatetag closebrace %}&to={% templatetag openbrace %}{% templatetag openbrace %}to{% templatetag closebrace %}{% templatetag closebrace %}
+      http://myservice.com/send.php?from={% templatetag openbrace %}{% templatetag openbrace %}from{% templatetag closebrace %}{% templatetag closebrace %}&text={% templatetag openbrace %}{% templatetag openbrace %}text{% templatetag closebrace %}{% templatetag closebrace %}&to={% templatetag openbrace %}{% templatetag openbrace %}to{% templatetag closebrace %}{% templatetag closebrace %}{% templatetag openbrace %}{% templatetag openbrace %}quick_replies{% templatetag closebrace %}{% templatetag closebrace %}{% templatetag closebrace %}
 
     %p
       -blocktrans trimmed
         If using POST or PUT, you can specify the body of the request using the same variables.
 
+    %p
+      %trans The variable
```
Comment (rowanseymour) [inline templates/channels/types/external/claim.haml]: shouldn't this be `-trans` ? please include a screenshot so we can see how this looks

Comment (rowanseymour) [inline templates/channels/types/external/claim.haml]: shouldn't this be -trans ? please include a screenshot so we can see how this looks

Comment (allancslima) [reply templates/channels/types/external/claim.haml]: I put the entire text inside a `trans` tag:

image

Comment (allancslima) [reply templates/channels/types/external/claim.haml]: I put the entire text inside a `trans` tag:

image

**Pull request merged**